addons/web_editor/static/lib/odoo-editor/src/style.css

```diff
@@ -328,3 +328,109 @@ i.oe-commandbar-commandImg {
     border-radius: 2px;
     padding: 0.3em 0.6em;
 }
+
+/* QWEB */
+
+.odoo-editor t,
+.odoo-editor [t-if],
+.odoo-editor [t-elif],
+.odoo-editor [t-else],
+.odoo-editor [t-foreach] {
+    background-color: rgba(0, 0, 102, 0.1) !important;
+}
+.odoo-editor t,
+.odoo-editor [t-esc],
+.odoo-editor [t-out],
+.odoo-editor [t-raw] {
+    border-radius: 2px;
+}
+.odoo-editor [t-esc],
+.odoo-editor [t-out],
+.odoo-editor [t-raw] {
+    background-color: rgb(36 154 255 / 16%) !important;
+}
+.odoo-editor [t-esc]:empty::before {
+    content: attr(t-esc);
+}
+.odoo-editor [t-raw]:empty::before {
+    content: attr(t-raw);
+}
+.odoo-editor [t-out]:empty::before {
+    content: attr(t-out);
+}
+.odoo-editor t[t-set] {
+    display: none;
+}
+.odoo-editor t[data-oe-t-inline] {
+    display: inline;
+}
+.odoo-editor t:not([data-oe-t-inline]) {
+    display: block;
+}
+.odoo-editor t[data-oe-t-inline]:not([data-oe-t-group-active]) {
+    display: unset;
+}
+.odoo-editor [data-oe-t-group]:not([data-oe-t-group-active]) {
+    display: none !important;
+}
+.odoo-editor [data-oe-t-group][data-oe-t-selectable] {
+    outline: 1px dashed rgba(0, 0, 102, 0.4) !important;
+}
+.oe-qweb-select {
+    position: absolute;
+    z-index: 1051;
+    background-color: white;
+}
+
+/* QWEB */
```

Comment:
Unless I missed something, all "QWEB"-related CSS lines are present twice in this file?
L332-382 vs L385-L436?

Reply:
Yes indeed. Corrected.